test/verify/check-sosreport

```diff
@@ -33,7 +33,11 @@ class TestSOS(testlib.MachineCase):
         super().setUp()
         m = self.machine
 
-        if m.image.startswith("debian") or m.image.startswith("ubuntu"):
+        # Newer Debian and Ubuntu systems have sosreport in /var/tmp now
+        # https://github.com/sosreport/sos/pull/4090
+        if (m.image.startswith("debian") and not m.image.startswith("debian-testing")) or (
```

Comment:
This should be simplified to `m.image in ["debian-trixie", "ubuntu-2204", "ubuntu-2404"]`. It's better to have the explicit "old" versions in the code, as then we can grep for them when finding outdated special cases when we e.g. drop the trixie or 22.04 image.